front/lib/plans/subscription.ts

```diff
@@ -393,6 +408,33 @@ export const getCheckoutUrlForUpgrade = async (
   };
 };
 
+export async function isStripeSubscriptionOnProPlan(
```

Comment:
No need to export here?

Reply:
I'll be using it on another file on my next PR!

Reply:
Actually not this one, adding a commit!